src/EnergyPlus/DXCoils.cc

```diff
@@ -9319,6 +9401,9 @@ void CalcDoe2DXCoil(EnergyPlusData &state,
     // If used in a heat pump, the value of MaxOAT in the heating coil overrides that in the cooling coil (in GetInput)
     if (CompAmbTemp < thisDXCoil.MaxOATCrankcaseHeater) {
         CrankcaseHeatingPower = thisDXCoil.CrankcaseHeaterCapacity;
+        if (thisDXCoil.CrankcaseHeaterCapacityCurveIndex > 0) {
+            CrankcaseHeatingPower *= Curve::CurveValue(state, thisDXCoil.CrankcaseHeaterCapacityCurveIndex, CompAmbTemp);
+        }
```

Comment:
This code could have probably been added to ReportDXCoil, instead of each calc funtion. It's OK as is, just noting an alternate way to account for the curve value.

Reply:
yeah, I do see a lot of places with this type of calculation.